containers/podman-desktop
# fix: restoring padding in settings screen headers

- The settings pages header titles were stripped of their vertical padding, that has been restored in this commit

### What does this PR do?

Fixes the vertical padding around the header titles in the various settings screens. It becomes more noticeable an issue with pending PR https://github.com/containers/podman-desktop/pull/2223 , see how "Resources" touches the bottom of the titlebar here?

![Screenshot from 2023-05-01 09-26-30](https://user-images.githubusercontent.com/799683/235457994-6c7a2503-cd01-4ea8-b50d-0b68d198dbb0.png)


### Screenshot/screencast of this PR

![Screenshot from 2023-05-01 09-20-02](https://user-images.githubusercontent.com/799683/235457435-5469d585-a8ba-43bf-beca-14aa6d6ed211.png)
![Screenshot from 2023-05-01 09-19-53](https://user-images.githubusercontent.com/799683/235457436-c7c6c67f-3f00-473b-8b34-f70dc0be06bc.png)

### What issues does this PR fix or reference?

I don't think we filed one for this.

### How to test this PR?

Run, click around the app with particular attention to pages under the main nav > Settings and make sure the titles are lined up nicely.
